### DISCUSSION TITLE
The Funny Looking Demolition Engine Build

### DISCUSSION THREAD

**Discussion by MemTheBun** [DEMOLITIONDON96/Demolition-Engine#98] [General (Memes)]
![image](https://user-images.githubusercontent.com/92312822/168404919-944e47de-9fd5-4d98-8fb4-20b1d361383e.png)

**Comment by MemTheBun** [comment]
thats how i name my demolition engine builds lol

**Comment by ghost** [comment]
Want me to change the YML file to look like that?

**Reply by ThatOneSillyGuyDon** [reply]
> Want me to change the YML file to look like that?

# no, don’t

**Reply by ghost** [reply]
> > Want me to change the YML file to look like that?
> 
> # no, don’t

Can we do it on April Fools?

**Reply by itsDARK768** [reply]
Thats in 8 months dude, fnf would be dead by then due to its community.

**Reply by ghost** [reply]
> Thats in 8 months dude, fnf would be dead by then due to its community.

No.
FNF Week 8 will drop then because it takes fucking forever.
And maybe Karlson???

**Reply by sumitshinde-84** [reply]
hii

**Reply by sumitshinde-84** [reply]
> > > Want me to change the YML file to look like that?
> > 
> > 
> > # no, don’t
> 
> Can we do it on April Fools?

**Comment by Theoyeah** [comment]
Were 2023 and FNF isn't ded yet sooooo
